sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -183,6 +183,14 @@ public enum AccountKind
         BlobStorage = 1,
         StorageV2 = 2,
     }
+    public partial class AppendBlobOpenWriteOptions
```

Comment:
`AppendBlobOpenWriteOptions`, `BlockBlobOpenWriteOptions` and `PageBlobOpenWriteOptions` all share `BufferSize`, `Overwrite`, and `ProgressHandler`.  Should we consider creating a parent class for them?

Reply:
Added in next iteration.